### PR TITLE
Add env vars

### PR DESCRIPTION
**Pull request by asachs01**
## Pull Request Checklist

**Is this in reference to an existing issue?**

#### General

- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [x] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [ ] RuboCop passes

- [ ] Existing tests pass

#### New Plugins

- [ ] Tests

- [ ] Add the plugin to the README

- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

#### Purpose
Adds the ability to use environment variables so that the plugin is more suited to the Sensu Go workflow.

### CODE REVIEW

**Comment by asachs01** [comment]
@jspaleta @cwjohnston tested this and is working w/ env vars. Can I get a 👍 / 👎 on this?

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+## [2.0.1] - 2019-06-10
+### Added
+- Adds ability to use environment variables.
```
Comment (cwjohnston) [inline CHANGELOG.md]: I think this needs to be more descriptive. Ability to use environment variables for what?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+## [2.0.1] - 2019-06-10
```
Comment (cwjohnston) [inline CHANGELOG.md]:
```suggestion
```

Let's avoid bumping the version in a PR like this, as I believe that change needs to come at the time of cutting the release.

**Review by cwjohnston — CHANGES_REQUESTED** [review]

**Comment by asachs01** [comment]
@cwjohnston should be 👍 now. Not sure how that Gem version bump got in there.

**Comment by jspaleta** [comment]
the .bonsai.yml filter has a mistake in it.   platform_version  not platform_release.  I'll fix that before cutting a new release as I was the one that introduced that bug in the example bonsai.

**Comment by jspaleta** [comment]
okay i fixed the platform_reversion oops in the bonsai.yml in the master branch, you might need to rebase or merge in from master ot fix the small conflict

**Comment by asachs01** [comment]
@cwjohnston this appears to be all good after looking at the main bonsai & the branch's. Can I get a 👍

**Review by cwjohnston — APPROVED** [review]

File: .bonsai.yml [hunk 3]
```diff
@@ -60,4 +60,4 @@ builds:
   -  "entity.system.os == 'linux'"
   -  "entity.system.arch == 'amd64'"
   -  "entity.system.platform_family == 'debian'"
-  -  "entity.system.platform_version.split('.')[0] == '9'"
+  -  "entity.system.platform_version.split('.')[0] == '9'"
```
Comment (majormoses) [inline .bonsai.yml]: why are we killing a newline?

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+### Added
+- Adds ability to use environment variables in Sensu Go.
```
Comment (majormoses) [inline CHANGELOG.md]: lets list out the env vars we enabled, this is not specific to `sensu-go` you can absolutely use ENV vars in the context of "sensu ruby".

File: .bonsai.yml (hunk 3 above)
Comment (asachs01) [reply .bonsai.yml]: I didn't realized I'd killed that off.

File: CHANGELOG.md (hunk 4 above)
Comment (asachs01) [reply CHANGELOG.md]: 👍

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-victorops/versions/2.1.0

**Comment by majormoses** [comment]
@jspaleta @cwjohnston looks like this repo needs the github token added for bonsai asset deployment: https://travis-ci.org/sensu-plugins/sensu-plugins-victorops/jobs/603094982#L1234

**Pull request merged**